tensorboard/tools/mirror_urls_test.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Check that all TensorFlow build mirror URLs resolve, and that we're
+# not using any legacy Bazel mirror URLs.
+#
+# This does not check that every URL in a build/workspace file is
+# properly mirrored.
+#
+# This test requires an Internet connection.
+
+set -eu
+
+if ! [ -f WORKSPACE ]; then
+    printf >&2 'fatal: no WORKSPACE file found (are you at TensorBoard root?)\n'
+    exit 2
+fi
+
+unresolved_urls_file=
+bazel_urls_file=
+cleanup() {
+    rm "${unresolved_urls_file}" || true
+    rm "${bazel_urls_file}" || true
+}
+trap cleanup EXIT
+
+check_urls_resolve() {
+    unresolved_urls_file="$(mktemp)"
+    # shellcheck disable=SC2016
+    check_cmd='curl -sfL "$1" >/dev/null || printf "%s\n" "$1"'
+    # shellcheck disable=SC2016
+    exclude='${version}'  # as in `ci/download_bazel.sh`
```

Comment:
Can we just exclude the specific ci/download_bazel.sh file itself?  That seems less brittle.

Reply:
Sure; done.